src/io/aviso/taxi_toolkit/fixtures.clj

```diff
@@ -0,0 +1,74 @@
+(ns io.aviso.taxi-toolkit.fixtures
+  "General clojure.test purpose fixtures for running selenium/webdriver tests.
+  Please note that this namespace relies on webdriver-remote library."
+  (:require [clj-webdriver.taxi :as taxi]
+            [clj-webdriver.remote.server :as webdriver-remote])
+  (:import (java.util.concurrent TimeUnit)))
+
+(defn webdriver-remote-fixture
```

Comment:
Could this fn be somehow separated in two pieces, one general for setting up a driver, timeouts etc., and the other one that would build on top of that with remote session stuff?